src/ui/server/controllers/insurance/business/company-details/validation/company-details/rules/alternative-trading-name.ts

```diff
@@ -0,0 +1,30 @@
+import { ERROR_MESSAGES } from '../../../../../../../content-strings';
+import FIELD_IDS from '../../../../../../../constants/field-ids/insurance/business';
+import { RequestBody } from '../../../../../../../../types';
+import emptyFieldValidation from '../../../../../../../shared-validation/empty-field';
+
+const {
+  YOUR_COMPANY: { ALTERNATIVE_TRADING_NAME: FIELD_ID, TRADING_NAME },
+} = FIELD_IDS;
+
+const {
+  EXPORTER_BUSINESS: { [FIELD_ID]: ERROR_MESSAGE },
+} = ERROR_MESSAGES.INSURANCE;
+
+/**
+ * validates alternative trading name field
+ * checks if response has been provided
+ * @param {RequestBody} responseBody
+ * @param {object} errors
+ * @returns {object} errors
+ */
+const alternativeTradingName = (responseBody: RequestBody, errors: object) => {
+  // if TRADING_NAME radio is yes then check validation
+  if (responseBody[TRADING_NAME] === 'true') {
```

Comment:
Could use `FIELD_ID` here, also not sure if we need  the comment? The line explains itself IMO(?)

Reply:
FIELD_ID is being used by alternative trading name, left the comment incase anyone misses that the previous field (trading name) has to be set to true for validation to run.  Can remove

Reply:
I'm a bit confused why the file is called "alternative trading name", but the function is checking `TRADING_NAME` and returning an error message via field ID `ALTERNATIVE_TRADING_NAME`

Reply:
its a conditional field so validation should only run if trading name is set to true.  Else validation will always run and wont be able to submit the form if trading name is false

Reply:
Shouldn't the radio button be called `ALTERNATIVE_TRADING_NAME`?

Because we're asking "if you have an alternative trading name - what is the name"

I.E if `ALTERNATIVE_TRADING_NAME` is true, we need a `TRADING_NAME`.

Seems its currently named the other way around?